cmd/kops/create_cluster.go

```diff
@@ -80,6 +81,8 @@ func init() {
 	cmd.Flags().StringVar(&createCluster.DNSZone, "dns-zone", "", "DNS hosted zone to use (defaults to last two components of cluster name)")
 	cmd.Flags().StringVar(&createCluster.OutDir, "out", "", "Path to write any local output")
 	cmd.Flags().StringVar(&createCluster.AdminAccess, "admin-access", "", "Restrict access to admin endpoints (SSH, HTTPS) to this CIDR.  If not set, access will not be restricted by IP.")
+
+	cmd.Flags().BoolVar(&createCluster.NoAssociatePublicIP, "no-associate-public-ip", false, "Specify --no-associate-public-ip to disable association of public IP for master ASG and nodes.")
```

Comment:
Actually, a better idea... what about adding a default value to `func (r *InstanceGroupRegistry) Find(name string) (*InstanceGroup, error) {` in registry.go.

The cleanest way might be to create a method on InstanceGroup called `SetDefaults` or similar, and then call it from Find in registry.go, and also in populate_instancegroup_spec.go.

We essentially want a field with a default value of true, but I don't know of an easier way.  And we do something similar in k8s.